packages/fondue/src/components/RichTextEditor/serializer/utils/serializeNodeToHtmlRecursive.ts

```diff
@@ -126,7 +126,7 @@ const MapNodeTypesToHtml: { [key: string]: ({ ...args }: Arguments) => string }
             getLiStyles(node, styles),
```

Comment:
Also needs to be fixed within the checklist serialization

Reply:
Thanks, fixed